src/helpers/eventBridge.ts

```diff
@@ -162,6 +168,30 @@ export default class EventBridge {
     return result;
   }
 
+  async getAllEvents(): Promise<EventBridgeEvents> {
+    let allEventsFound = false;
+    const allEventBridgeEvents: EventBridgeEvents = {};
+
+    while (!allEventsFound) {
+      try {
+        const lastEventBridgeEvents = await this.getEvent();
+
+        if (!lastEventBridgeEvents || !lastEventBridgeEvents.Messages) {
+          return allEventBridgeEvents;
+        }
```

Comment:
```suggestion
        const lastEventBridgeEvent = await this.getEvent();

        if (!lastEventBridgeEvent || !lastEventBridgeEvent.Messages) {
          return allEventBridgeEvents;
        }
```

Just to make it super obvious that the returned valued from `this.getEvent()` represents a single event only